README.md

```diff
@@ -0,0 +1,58 @@
+# Technical Test
+
+## Introduction
+
+This technical test makes use of the dvd rentals Postgres database. The entity–relationship model has been provided to you to better understand what is contained in this database.
+
+![ER Model](dvd-rental-db-diagram.png)
+
+The database can be hosted locally using Docker, allowing you to execute queries against it.
+
+We have set a series of tasks for you to complete as outlined below. For each question, please copy the queries you have written to a document (Google Docs, Markdown etc) and share it with us (email, Github etc).
+
+While the data set provided is relatively small, please attempt to write performant queries.
+
+## Set Up
+
+### Prerequisites
+
+- Docker installed on your local machine.
+- Saved `technical_test` folder containing docker compose file and dvdrental folder to your machine.
+
+### Hosting the database
+
+Execute:
+
+- `docker compose up`
+- `docker exec -it pg_container bash`
+  - This will give access to the container `pg_container`
+- Set the database password inside the container; `set "PGPASSWORD=root"`
+- Load the database; `pg_restore -U postgres -d dvdrental dvdrental`
+
+### Connecting to the database
+
+There are many options to connect to the database including:
+
+- psql via CLI
+- SQL editor such as DBeaver
+- dbt
+
+Please use whatever you feel most comfortable with. The connection details can be found in the `docker_compose.yml` file. The host name will likely be `localhost`.
+
+## Questions
```

Comment:
I wonder if it's worth stating you don't have to answer all these questions, just do the ones you feel comfortable with or show us your workings on your attempts for each question even if you didn't quite find the answer. A bit like a Maths GCSE exam 😇